server/server/common/queries.py

```diff
@@ -462,6 +461,37 @@
 '''
 
 GLOSSARY = '''
-FOR gloss IN glossary
-    RETURN KEEP(gloss, ['glossword', 'description'])
+LET glossaryitem = (
+    FOR dictionary IN dictionary_full
+        FILTER dictionary.dictname == "gloss"
+        RETURN KEEP(dictionary, "word", "text")
+    )
+    
+RETURN glossaryitem
+'''
+
+ADJACENT = '''
+LET allitems = (
+    FOR dictionary IN dictionary_full
+        RETURN (dictionary.word)
+    )
+
+LET uniqueterms = (
+    RETURN UNIQUE(allitems)
+    )
+
+LET sorteditem = (
+    FOR items in uniqueterms[0]
+        SORT items ASC
+        RETURN (items)
+    )
+
+LET searchword = (
```

Comment:
ditto.
It also seems this is not a word, but a number, so the variable name is confusing.